### ISSUE TITLE
Private registries

### ISSUE THREAD

**Issue by RongxinZhang** [CenturyLinkLabs/imagelayers#27]
#### Issues:
- It looks like this can only support public registries. OR am I wrong?
- Is there away to read from a local docker image

**Comment by RongxinZhang** [comment]
@SydOps Any thoughts?

**Comment by SydOps** [comment]
the question has been answered and cloded 
#24

**Comment by ckleban** [comment]
Yes, are this time, we are support public registries.  However, adding support for private registries or uploaded/local docker files might be something someone might want to add.

**Comment by ismarslomic** [comment]
We are running private docker registry and would love to be able to run imagelayers locally.
